<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesWnd</class>
 <widget class="QDialog" name="PreferencesWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../HomeTheatre.qrc">:/icons/Resources/icon-settings.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Segoe UI Semilight</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Preferences</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>578</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../HomeTheatre.qrc">
          <normaloff>:/icons/Resources/icon-close.png</normaloff>:/icons/Resources/icon-close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="btnFolders">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Folders</string>
        </property>
        <property name="icon">
         <iconset resource="../../HomeTheatre.qrc">
          <normaloff>:/icons/Resources/icon-folder.png</normaloff>:/icons/Resources/icon-folder.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSearch">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
        <property name="icon">
         <iconset resource="../../HomeTheatre.qrc">
          <normaloff>:/icons/Resources/icon-folder.png</normaloff>:/icons/Resources/icon-folder.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Folders to scan</string>
         </property>
        </widget>
        <widget class="QListWidget" name="foldersList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>256</width>
           <height>231</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="btnAddDir">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>40</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnRemoveDir">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>80</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnRescan">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>280</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Rescan</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnRebuild">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Rebuild Library</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../HomeTheatre.qrc"/>
 </resources>
 <connections/>
</ui>
